Sources/Common/DataModel/ImageData/index.js

```diff
@@ -233,6 +234,21 @@ function vtkImageData(publicAPI, model) {
     return bounds;
   };
 
+  // Get array extents as float / spatial coordinates
+  // without scaling for voxel/pixel spacing.
+  publicAPI.getSpatialExtent = () => {
```

Comment:
@floryst @martinken how do you like the `getSpatialExtent` name? Does this make sense here?

Reply:
`getGridBounds` is another suggestion that might work.

Reply:
I believe we can stick with `getSpatialExtent` for now. Can be easily updated in the future.